README.md

```diff
@@ -133,14 +137,20 @@ console.log(inspect(9));
 ### Objects
 
 ```js
-console.log(inspect({
-  obj: {
-  a: [ { foo: 'bar', baz: { qux: 'fez'}      }],
-  c: 'd', e: 'f',
-  g: function  (one, two){return one + two},
-  h: {a: 'b', c: 'd'},
-  regex: /^foo(?=bar)/g }
-}));
+console.log(
+  inspect({
+    obj: {
+      a: [{foo: 'bar', baz: {qux: 'fez'}}],
+      c: 'd',
+      e: 'f',
+      g: function(one, two) {
+        return one + two;
+      },
+      h: {a: 'b', c: 'd'},
+      regex: /^foo(?=bar)/g
+    }
+  })
+);
```

Comment:
I guess we need to add `prettier-ignore` in some places.. Let me know where and I'll do ti.